src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -1698,5 +1820,26 @@ def reshape_to_vivarium_format(df, location):
     df = vi_utils.split_interval(df, interval_column="age", split_column_prefix="age")
     df = vi_utils.split_interval(df, interval_column="year", split_column_prefix="year")
     df = vi_utils.sort_hierarchical_data(df)
-    df.index = df.index.droplevel("location")
+    # TODO: delete me
+    # df.index = df.index.droplevel("location")
     return df
+
+
+def fetch_subnational_ids(location: str) -> List[int]:
+    location_id = utility_data.get_location_id(location)
+    location_metadata = gbd.get_location_path_to_global()
+    subnational_location_metadata = location_metadata.loc[
+        (location_metadata["path_to_top_parent"].apply(lambda x: str(location_id) in x))
+        & (location_metadata["location_id"] != location_id)
+    ]
+    subnational_location_ids = subnational_location_metadata["location_id"].tolist()
+    return subnational_location_ids
+
+
+def get_national_location_id(location_id: int) -> int:
```

Comment:
Where is this used?

Reply:
This currently is not being used however there will likely be a use case in the future where we weight subnational data with national weights. We have raw data csvs at the national level we would use for this and this is what this function would do.